renovate.json

```diff
@@ -35,9 +35,7 @@
     "gomodUpdateImportPaths",
     "gomodTidy"
   ],
-  "rebaseWhen": "behind-base-branch",
```

Comment:
maybe we need to set this to "conflicted" excplicitely. Not sure about the default value